mhlo_builder/csrc/mlir/mhlo/builder/element_wise_binary.h

```diff
@@ -28,7 +28,26 @@ static constexpr const char kCompare_GT[] = "GT";
 static constexpr const char kCompare_LE[] = "LE";
 static constexpr const char kCompare_LT[] = "LT";
 
-mhlo::ComparisonDirection inline getComparisonDirectionFromString(
+chlo::ComparisonDirection inline getChloComparisonDirectionFromString(
+    std::string cmp) {
```

Comment:
`const std::string&` ? And why not receive `const char*`?

Reply:
done